scality_sproxyd_client/sproxyd_client.py

```diff
@@ -41,28 +41,35 @@ class SproxydClient(object):
 
     DEFAULT_LOGGER = logging.getLogger(__name__)
 
-    def __init__(self, endpoints, conn_timeout=10.0, proxy_timeout=3.0,
-                 logger=None):
+    def __init__(self, endpoints, conn_timeout=10.0, read_timeout=3.0,
+                 logger=None, **kwargs):
         '''Construct an `sproxyd` client
 
         This client connects in a round-robin fashion to online Sproxyd
         connectors provided in the `endpoints` list.
 
-        If `conn_timeout`, `proxy_timeout` or `logger` are not provided, a
+        If `conn_timeout`, `read_timeout` or `logger` are not provided, a
         default value will be used.
 
         :param endpoints: URLs where Sproxyd connectors accept 'by path' queries
         :type endpoints: iterable of `str` or `urlparse.ParseResult`
         :param conn_timeout: Connection timeout
         :type conn_timeout: `float`
-        :param proxy_timeout: Proxy timeout
-        :type proxy_timeout: `float`
+        :param read_timeout: Proxy timeout
+        :type read_timeout: `float`
         :param logger: Logger used by methods on the instance
         :type logger: `logging.Logger`
         '''
 
         self._conn_timeout = conn_timeout
-        self._proxy_timeout = proxy_timeout
+
+        # `read_timeout` used to be named `proxy_timeout`. We maintain
```

Comment:
Do we need backwards compatibility for a library which hasn't had any release? Using full-fledged `**kwargs` feels a bit gross.